schema/2.0.0/corpora_schema.md

```diff
@@ -441,7 +441,7 @@ Curators MUST annotate the following columns in the `obs` dataframe:
           <tbody>
             <tr>
               <td>Tissue</td>
-              <td>MUST be an UBERON term<br>(e.g. <a href="http://purl.obolibrary.org/obo/UBERON_0008930"><code>"UBERON:0008930"</code></a> for a <i>sematosensory cortex</i> tissue sample)</td>
+              <td>STRONGLY RECOMMENDED to be an UBERON term<br>(e.g. <a href="http://purl.obolibrary.org/obo/UBERON_0008930"><code>"UBERON:0008930"</code></a> for a <i>sematosensory cortex</i> tissue sample)</td>
```

Comment:
I'm lacking the context for this change. Why is it no longer a MUST?

Reply:
Per the linked comment thread, we now support both UBERON and CL and cannot programmatically discern between _Tissue_ and  _Enriched, Sorted, or Isolated Cells from a Tissue_ to enforce (MUST) UBERON for Tissue.

Reply:
I see (sorry!).
Kicking around another option in my head (spoiler alert: I vote 'nay' on it) - add an appendage to the bottom case of `Enriched, Sorted,or Isolated Cells from a Tissue`. You could then enforce that the lack of an appendage MUST be an UBERON term.
This could also help flag for the user that data from the CD3+ kidney cells examples should not be taken in the same light as unenriched kidney tissue data (currently, each would be tissue:kidney).

However, the tissue field is one where we do not see much issue with. The proposed changes still allow for a submitter to choose the most appropriate term, whether it be at a cellular (CL) or multi-cellular (UBERON) level